test/regressionTest.m

```diff
@@ -0,0 +1,57 @@
+function regressionTest(commitBefore,commitAfter)
```

Comment:
It can be used for both. 
But mostly this is to test whether a regression is present between commit X and Y. I wouldn't hardcode the `develop` branch (at most as the default argument when for `commitBefore`).